extensions/wikia/Insights/Insights.i18n.php

```diff
@@ -8,17 +8,25 @@
 	'insights-landing-title' => 'Welcome to Insights',
 	'insights-landing-lead' => 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean quis consectetur lacus, vel hendrerit urna. In id libero erat. Vestibulum viverra, tortor vel commodo finibus, augue nulla elementum ex, sit amet sodales augue lacus ut augue. Cras blandit ex sed sodales scelerisque. Sed ornare semper nulla, quis ullamcorper nisi eleifend eu. Ut quis mauris libero. Sed cursus mauris quis augue aliquet malesuada. Nunc sed tincidunt arcu.',
 	'insights-last-edit' => 'Last edited by $1, $2',
-	'insights-list-uncategorized-subtitle' => 'Uncategorized pages',
-	'insights-list-uncategorized-description' => 'Uncategorized pages is the most awesome feature ever!',
-	'insights-list-withoutimages-subtitle' => 'Pages without images',
-	'insights-list-withoutimages-description' => 'The pages need their images!',
-	'insights-list-deadend-subtitle' => 'Deadend pages',
-	'insights-list-deadend-description' => 'A list of places you don\'t want to end up in. They are dead. They are end. They are your nightmare.',
-	'insights-list-wantedpages-subtitle' => 'Wanted pages',
-	'insights-list-wantedpages-description' => 'Somebody wants these pages REALLY BAD!',
-	'insights-notification-message' => 'Nice job! Where would you like to go now?',
-	'insights-notification-list-button' => 'Show Insights page',
-	'insights-notification-next-item-button' => 'Fix next article',
+	'insights-list-subtitle-uncategorizedpages' => 'Uncategorized pages',
+	'insights-list-description-uncategorizedpages' => 'Uncategorized pages is the most awesome feature ever!',
+	'insights-list-subtitle-withoutimages' => 'Pages without images',
+	'insights-list-description-withoutimages' => 'The pages need their images!',
+	'insights-list-subtitle-deadendpages' => 'Deadend pages',
+	'insights-list-description-deadendpages' => 'A list of places you don\'t want to end up in. They are dead. They are end. They are your nightmare.',
+	'insights-list-subtitle-wantedpages' => 'Wanted pages',
+	'insights-list-description-wantedpages' => 'Somebody wants these pages REALLY BAD!',
+	'insights-notification-message-inprogress-uncategorizedpages' => 'This page needs categories added. Let’s fix it.',
```

Comment:
Please use the strings that Trevor has created - https://wikia-inc.atlassian.net/browse/CE-1627